cmd/cdi-controller/controller.go

```diff
@@ -145,7 +145,14 @@ func start(ctx context.Context, cfg *rest.Config) {
 		klog.Fatalf("Error building extClient: %s", err.Error())
 	}
 
-	mgr, err := manager.New(config.GetConfigOrDie(), manager.Options{})
+	opts := manager.Options{
+		LeaderElection:             true,
+		LeaderElectionNamespace:    namespace,
+		LeaderElectionID:           "cdi-controller-leader-election-helper",
```

Comment:
Maybe use the configMapName const from leaderelection.go? Or rename that something more fitting?

Reply:
I would like to jut delete `leaderelection.go` once we cut over

Reply:
Sure, maybe move the const in here then?